source/adminguide/networking/virtual_private_cloud_config.rst

```diff
@@ -1444,6 +1444,26 @@ Editing, Restarting, and Removing a Virtual Private Cloud
    |restart-vpc.png|
 
 
+Working with Domain VPCs
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+The functionality of domain VPCs allows operators to aggregate multiple
+tiers from distinct users on the same VPC, reducing the number of virtual
+routers necessary in the environment, and consequently, decreasing the
+amount of public IP addresses consumed. All tiers added to the VPC share
+the same VR, but each one has their own broadcast domain and features
+implemented by the VPC, such as DHCP, NAT, and so on.
+
+In order to utilize this functionality, a new tier must be included to an
+existing VPC by inputing the respective data for the account and the VPC
+on the **'createNetwork'** API. It is important to note that, in order
+for a tier of a different account to be created on the VPC, the account
+that creates the tier must have access to both the account that owns the
+VPC and the account that owns the tier. The owner of the VPC must also
+have access to the account that owns the network, however, the opposite
```

Comment:
What is the network here? Is it the same as the tier?

Reply:
Yes, they are the same. 
Earlier in the document, during the introduction of the components the tiers were introduced as `Network tiers`, so I have used both words synonymously to avoid overusing the word tier. However, I can change all mentions of network back to tier if it is affecting the readability.

Reply:
@GaOrtiga could you please change it to network tiers as in vpc UI and docs we are referring it as network-tier

Reply:
I think if it is clear that we are constantly talking about network-tier, I don't think we need an extra example. But please avoid the use of either "network" and possible "tier". We are not writing proza and though prozaic value is never wrong, in this case it may be somewhat confusing indeed.